src/main/java/DecoratorForString.java

```diff
@@ -0,0 +1,46 @@
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.jsoup.select.Elements;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+public class DecoratorForString implements Links{
+
+    protected String url;
+
+    public DecoratorForString(String url) {
+        this.url = url;
+    }
+
+    private List<String> prepareLinks() {
```

Comment:
@PavelNejinskiy please follow to this blog while choosing name for the method/class
http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html

Lets rename this method to just `links`